src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -334,23 +328,23 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
       stateObj.attributes.target_temp_low &&
       stateObj.attributes.target_temp_high
     ) {
-      if (e.handle.index === 1) {
+      if (e.detail.low) {
```

Comment:
The logic do determine which value to set is not the same as in `_dragEvent`. Why?

Reply:
Results are the same, but the way to get there is not.